extensions/amp-animation/0.1/amp-animation.js

```diff
@@ -306,14 +333,29 @@ export class AmpAnimation extends AMP.BaseElement {
       return null;
     }
 
-    return this.createRunner_(opt_args).then(runner => {
-      this.runner_ = runner;
-      this.runner_.onPlayStateChanged(this.playStateChanged_.bind(this));
-      this.setupScrollboundAnimations_();
+    return this.maybeCreateRunner_(opt_args).then(() => {
       this.runner_.start();
     });
   }
 
+  /**
+   * Creates the runner but animations will not start.
+   * @param {?JsonObject=} opt_args
+   * @return {!Promise}
+   * @private
+   */
+  maybeCreateRunner_(opt_args) {
```

Comment:
What are potential consequences of this? Is it strictly necessary? Is it to be able to seek right away? Maybe it's better to `animate()` right before the first `seek`? Feels like a biggish change from before, but maybe it's not.